src/app/favourites/_components/FavouritesFull.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+
+import Button from '@/components/ui/Button'
+import FavElement from './FavElement'
+
+import { IProduct } from '@/models/Products'
+import Link from 'next/link'
+import { useFavouritesStore } from '@/store/favStore'
+
+export default function FavouritesFull() {
+  const { favourites } = useFavouritesStore()
+
```

Comment:
U should fetch products from backend (sending ids to the endpoint) and then render it on the page

Reply:
fixed